main.js

```diff
@@ -1,6 +1,35 @@
+const projectData = [
+  {
+    id: 1,
+    name: ['Multi Post Stories', 'Keeping track of hundreds  of components website'],
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    featured: 'project-1 featured',
+    images: {
+      mobile: ['images/Snapshoot Portfoliopopup.png'],
+      desktop: ['images/Snapshoot Portfoliodeskpop.png'],
+    },
+    technologies: ['html', 'Bootstrap', 'Ruby on Rails'],
+    liveVersionLink: '#',
+    linkToSource: '#',
+  },
```

Comment:
[OPTIONAL] 👍🏽 

- I think it will be cool to add `see project` to all cards in your portfolio and populate them with data so that subsequently when you build other projects, you can always add them to a card on your portfolio. Also, with the link to source(github) and live demo 🤗

Reply:
thank you for the suggestion i will do them